ChequeWriter.csproj

```diff
@@ -10,6 +10,7 @@
 	  <Platforms>AnyCPU;x64</Platforms>
 	  <EnableNETAnalyzers>True</EnableNETAnalyzers>
 	  <AnalysisLevel>latest-recommended</AnalysisLevel>
+    <EnableWindowsTargeting>true</EnableWindowsTargeting>
```

Comment:
Add an intent to line 13 to align vertically with above code block

Reply:
in my editor line 13 is already indented, not sure why in github shows without indent